modules/fts/pages/fts-advanced-index-dynamic-fields.adoc

```diff
@@ -0,0 +1,5 @@
+= Index Dynamic Fields
+ When checked, ensures dynamic fields are indexed.
```

Comment:
This doesn't quite explain what this is for, just puts the heading in a different format.
Must do better here.